Student Run/metroSign.h

```diff
@@ -0,0 +1,16 @@
+// headers
+#include "globals.h"
+#include "colors.h"
+#include "utils.h"
+
+void metroSign(void) {
+	//metrou box
+	drawRectangle(10, 50, 40, 80, white);
+
+	//"M" letter
+	glColor3f(0.0, 0.0, 1.0); //blue
```

Comment:
@VMadalina 
TODO: adauga culoarea Color blue(0,.0f, 0.0f, 1.0f) direct in metroSign.h (Exista o versiune mai avansata in colors.h, se rezolva la un refactor
)